src/Orleans/IDs/Interner.cs

```diff
@@ -246,10 +246,14 @@ private void InternCacheCleanupTimerCallback(object state)
                 }
             }
             long numRemoved = numEntries - internCache.Count;
-            if (numRemoved>0)
+            if (numRemoved > 0)
+            {
                 if (logger.IsVerbose) logger.Verbose(ErrorCode.Runtime_Error_100296, "Removed {0} / {1} unused {2} entries in {3}", numRemoved, numEntries, internCacheName, clock.Elapsed);
+            }
             else
+            {
                 if (logger.IsVerbose2) logger.Verbose2(ErrorCode.Runtime_Error_100296, "Removed {0} / {1} unused {2} entries in {3}", numRemoved, numEntries, internCacheName, clock.Elapsed);
```

Comment:
hmmm, not sure @dVakulen since that wouldn't make a lot of sense. If IsVerbose is false, then IsVerbose2 will certainly be false. I think that if numRemoved == 0 it was logging as Verbose2